bin/commands/runs.js

```diff
@@ -9,14 +9,21 @@ const archiver = require("../helpers/archiver"),
   utils = require("../helpers/utils"),
   fileHelpers = require("../helpers/fileHelpers"),
   syncRunner = require("../helpers/syncRunner"),
-  reportGenerator = require('../helpers/reporterHTML').reportGenerator;
+  reportGenerator = require('../helpers/reporterHTML').reportGenerator,
+  {initTimeComponents, markBlockStart, markBlockEnd, getTimeComponents} = require('../helpers/timeComponents');
 
 module.exports = function run(args) {
   let bsConfigPath = utils.getConfigPath(args.cf);
   //Delete build_results.txt from log folder if already present.
+  initTimeComponents();
+  markBlockStart('deleteOldResults');
   utils.deleteResults();
+  markBlockEnd('deleteOldResults');
 
+  markBlockStart('validateBstackJson');
   return utils.validateBstackJson(bsConfigPath).then(function (bsConfig) {
+    markBlockEnd('validateBstackJson');
```

Comment:
What if the validateBstackJson throws an error?

Reply:
we do not want time measurements for unsuccessful builds as of now.

Reply:
Is this in sync with the product?

Reply:
yes